### PR TITLE
Update for modern browsers

### PR DESCRIPTION
**Pull request by mislav**
- drop IE 11
- Babel 7 + babel-preset-github
- new eslint-plugin-github
- latest Flow
- update Karma to resolve npm audit warnings

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-/* @flow */
+/* @flow strict */
 
 class CustomElementElement extends HTMLElement {
   constructor() {
     super()
   }
 
   connectedCallback() {
-    this.textContent = '<custom-element>'
```
Comment (koddsson) [inline index.js]: Why was this changed?

Comment (mislav) [reply index.js]: Two reasons:

1. potentially confusing (there's no reason why this arbitrary string should look like a tag name
2. violates `github/unescaped-html-literal` from eslint-plugin-github
